firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionEvent.kt

```diff
@@ -31,6 +31,7 @@ internal data class SessionEvent(
 
   /** Information about the session triggering the event. */
   val sessionData: SessionInfo,
+  val applicationInfo: ApplicationInfo,
```

Comment:
Can this line have doc too? From iOS "Information about the running application."

Reply:
Done.